components/ProfilePage/EditAvatarDialog.js

```diff
@@ -222,7 +225,11 @@ function EditAvatarDialog({ userId, onClose = () => {} }) {
             />
           </div>
           {avatarType === 'OpenPeeps' ? (
-            <AvatarSelector avatarData={avatarData} onChange={setAvatarField} />
+            <AvatarSelector
+              avatarData={avatarData}
+              onChange={setAvatarField}
+              shuffle={shuffle}
```

Comment:
Suggest using the name `onShuffle` for this callback to follow the convention of React componentization.